core/sr-std/with_std.rs

```diff
@@ -14,6 +14,8 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
+pub use std::alloc;
```

Comment:
I thought we want to avoid the entire `alloc`? We do some whitelist in without-std, no?

Reply:
Yeah. On `no_std` we just export some of it.